src/parsehdr.c

```diff
@@ -436,6 +437,12 @@ cr_package_from_header(Header hdr,
                         pkg->obsoletes = g_slist_prepend(pkg->obsoletes, dependency);
                         break;
                     case DEP_REQUIRES:
+#ifdef ENABLE_LEGACY_WEAKDEPS
+                        if ( num_flags & RPMSENSE_MISSINGOK ) {
```

Comment:
nit: The rest of this codebase doesn't use space ` ` inside conditionals.

Reply:
This logic was adapted from the one used with the `RPMSENSE_STRONG` check further down in the code.